src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
 }
+
+header {
+  padding: 0 50px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav_list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+}
+
+.nav_item {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  list-style: none;
+}
+
+.nav_item:not(:last-child) {
+  margin-right: 20px;
+}
+
+header a {
```

Comment:
Avoid using tag names for styling. Instead, style elements using `.class`. You can use a class for the `a` elements inside the header to apply these styles.